webapp/components/file_attachment.jsx

```diff
@@ -120,7 +114,7 @@ class FileAttachment extends React.Component {
             var re3 = new RegExp('\\)', 'g');
             var url = fileUrl.replace(re1, '%20').replace(re2, '%28').replace(re3, '%29');
```

Comment:
Me too, and I'd rather not touch regexp and have the server horrendously crash in a few days :stuck_out_tongue: